bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_10/A5_10.java

```diff
@@ -18,20 +18,26 @@
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.enocean.internal.Helper;
 import org.openhab.binding.enocean.internal.eep.Base._4BSMessage;
 import org.openhab.binding.enocean.internal.messages.ERP1Message;
 import org.openhab.core.config.core.Configuration;
 import org.openhab.core.library.types.DecimalType;
 import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.OpenClosedType;
 import org.openhab.core.library.types.QuantityType;
-import org.openhab.core.library.types.StringType;
 import org.openhab.core.library.unit.SIUnits;
+import org.openhab.core.library.unit.Units;
 import org.openhab.core.types.State;
 import org.openhab.core.types.UnDefType;
 
 /**
  *
  * @author Daniel Weber - Initial contribution
+ * 
+ *         From A5_10_01 up to A5_10_0D temperature is given as a 8Bit value (range: 255..0).
+ *         Therefore higher values mean lower temperatures.
+ *         Temperature range 0..40.
```

Comment:
I can't make suggestions to existing code, but can you reformat this to:
```java
/**
 * From A5_10_01 up to A5_10_0D temperature is given as a 8Bit value (range: 255..0).
 * Therefore higher values mean lower temperatures.
 * Temperature range 0..40.
 *
 * @author Daniel Weber - Initial contribution
 */
```

This will ensure it will be correctly shown in Javadoc and IDE's.